scripts/installer/install.py

```diff
@@ -229,12 +230,12 @@
    print("Checking for required components...")
    # 1a. check to see if warfile is available
    warfile = "dataverse.war"
-   warfileVersion = None
+   warfileVersion = ""
    if not os.path.isfile(warfile):
       # get dataverse version from pom.xml
       tree = ET.ElementTree(file='../../pom.xml')
       for elem in tree.iter("*"):
-         if elem.tag == '{http://maven.apache.org/POM/4.0.0}version':
+         if elem.tag == '{https://maven.apache.org/POM/4.0.0}version':
```

Comment:
I'm just noting the conversation about this change at https://github.com/IQSS/dataverse/pull/6812#issuecomment-612100332